composer.json

```
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": ">=7.0.0",
+        "php": ">=7.1.0",
```

we should make sure we dont collide with php8

```suggestion
        "php": "^7.1.0",
```